client/lang/en.js

```diff
@@ -10,6 +10,6 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "LinkField.CONFIRM_DELETE": "Deleted link",
     "LinkField.DELETE_ERROR": "Failed to delete link",
     "LinkField.ADD_LINK": "Add Link",
-    "LinkField.CLEAR": "Clear"
+    "LinkField.Delete": "Delete"
```

Comment:
```suggestion
    "LinkField.DELETE": "Delete"
```
I would normally ignore this, but it's being referenced with all caps in the code so needs to be all caps here.